administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -246,9 +246,7 @@
 
 $input      = $this->app->input;
 $layout     = $input->get('layout', '', 'string');
-$aComponent = $input->get('acomponent', '', 'string');
-$aView      = $input->get('aview', '', 'string');
-$extension  = $input->get('extension', '', 'string');
+$component = $input->get('component', '', 'string');
 $rLanguage  = $input->get('referencelanguage', '', 'string') != null ? $input->get('referencelanguage', '', 'string') : '';
```

Comment:
this will be alwsys different than null because you set the default value as an empty string

why not just
$rLanguage  = $input->get('referencelanguage', '', 'string');